File: app/src/main/java/com/goldmines/JsonPojos/Explain.java

package com.goldmines.JsonPojos;

import javax.annotation.Generated;
import com.google.gson.annotations.SerializedName;

@Generated("net.hexar.json2pojo")
@SuppressWarnings("unused")
public class Explain {

    @SerializedName("Answers")
    private String mAnswers;
    @SerializedName("Media")
    private Media mMedia;
    @SerializedName("qid")
    private String mQid;

    public String getAnswers() {
        return mAnswers;
    }

    public void setAnswers(String answers) {
        mAnswers = answers;
    }

    public Media getMedia() {
        return mMedia;
    }

    public void setMedia(Media media) {
        mMedia = media;
    }

    public String getQid() {
        return mQid;
    }

    public void setQid(String qid) {
        mQid = qid;
    }

}

File: app/src/main/java/com/goldmines/jsonkotlinpojos/Explain.kt
package com.goldmines.jsonkotlinpojos


import com.goldmines.xmlparsing.MediaX
import com.google.gson.annotations.SerializedName

data class Explain(
        @SerializedName("Answers")
        val answers: String,
        @SerializedName("MediaX")
        val media: MediaX,
        @SerializedName("qid")
        val qid: String
)